Automattic/jetpack
# Don't consider user lifecycle status when repeating evaluation

- **Don't consider user lifecycle status when repeating evaluation**
- **changelog**





Fixes #

## Proposed changes:
* Show evaluation survey on "Redo" regardless user lifecycle status

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Follow the instructions from here (https://github.com/Automattic/jetpack/pull/38664) to get the treatment version
* Finish the evaluation, when you get recommendations
* Install and/or purchase new plugin to not be considered a new user
* Click context menu (three dots) and select "Redo" from the menu
* You should see evaluation survey again
